src/main/java/randoop/main/GenInputsAbstract.java

```diff
@@ -431,6 +447,14 @@ public GenInputsAbstract(
   public static boolean small_tests = false;
 
   /**
+   * Whether to use DigDog weighted sequences in sequence selection. Note that this weighting scheme
```

Comment:
We have removed the mentions of the name "DigDog".

Would it be more appropriate to explain our command-line additions in depth in the developer manual or here in the descriptions of the command-line options?